dotCMS/src/integration-test/java/com/dotcms/publishing/listener/SecurityKeyResetTest.java

```diff
@@ -0,0 +1,95 @@
+package com.dotcms.publishing.listener;
+
+import static com.dotcms.publisher.business.PublisherTestUtil.createEndpoint;
+import static com.dotcms.publisher.business.PublisherTestUtil.createEnvironment;
+import static org.mockito.Mockito.when;
+
+import com.dotcms.IntegrationTestBase;
+import com.dotcms.LicenseTestUtil;
+import com.dotcms.datagen.TestUserUtils;
+import com.dotcms.publisher.bundle.bean.Bundle;
+import com.dotcms.publisher.bundle.business.BundleAPI;
+import com.dotcms.publisher.business.PublisherAPI;
+import com.dotcms.publisher.endpoint.bean.PublishingEndPoint;
+import com.dotcms.publisher.environment.bean.Environment;
+import com.dotcms.repackage.org.apache.struts.Globals;
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.portlets.contentlet.model.Contentlet;
+import com.dotmarketing.portlets.languagesmanager.model.Language;
+import com.dotmarketing.util.Config;
+import com.liferay.portal.model.User;
+import com.liferay.portal.struts.MultiMessageResources;
+import com.liferay.portal.struts.MultiMessageResourcesFactory;
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class SecurityKeyResetTest extends IntegrationTestBase {
+
+     @BeforeClass
+     public static void prepare() throws Exception {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5677177)